docker/manage

```diff
@@ -68,7 +72,10 @@ usage() {
       stop - Stops the services.  This is a non-destructive process.  The containers
              are not deleted so they will be reused the next time you run start.
     
-      rm - Remove project containers and associated volumes.
+      down - Brings down the services and removes the volumes (storage).  The containers
```

Comment:
The `down` command is destructive, containers and volumes are deleted.  Please update documentation.